lib/Dancer2/Plugin/UnicodeNormalize.pm

```diff
@@ -6,18 +6,17 @@ package Dancer2::Plugin::UnicodeNormalize;
     $Dancer2::Plugin::UnicodeNormalize::VERSION = '0.033';
 }
 
-use Dancer2 0.14300;
 use Dancer2::Plugin;
 use Unicode::Normalize;
 
 on_plugin_import {
     my $dsl = shift;
-    my $settings = plugin_setting();
 
     $dsl->app->add_hook(
         Dancer2::Core::Hook->new(
             name => 'before',
             code => sub {
+                my $settings = plugin_setting;
```

Comment:
@xsawyerx Should this call work from a hook's coderef? Output from one of the tests:

```
t/04-undef.t ............. [t::lib::TestApp:26735] error @2016-01-06 10:03:02> Exception caught in 'core.app.before_request' filter: Hook error: No DSL object found at /home/fuzzix/projects/Dancer2-Plugin-UnicodeNormalize/.build/sK6duD46HG/blib/lib/Dancer2/Plugin/UnicodeNormalize.pm line 19.
 at /home/fuzzix/perl5/perlbrew/perls/perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/App.pm line 973. in /home/fuzzix/perl5/perlbrew/perls/perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/App.pm l. 540
[t::lib::TestApp:26735] error @2016-01-06 10:03:02> Route exception: Exception caught in 'core.app.before_request' filter: Hook error: No DSL object found at /home/fuzzix/projects/Dancer2-Plugin-UnicodeNormalize/.build/sK6duD46HG/blib/lib/Dancer2/Plugin/UnicodeNormalize.pm line 19.
 at /home/fuzzix/perl5/perlbrew/perls/perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/App.pm line 973.
 at /home/fuzzix/perl5/perlbrew/perls/perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/Role/Hookable.pm line 115. in /home/fuzzix/perl5/perlbrew/perls/perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/App.pm l. 1286
t/04-undef.t ............. 1/1 
#   Failed test 'Do not bail out on missing/undef optional params'
#   at t/04-undef.t line 14.
#          got: '<!DOCTYPE html>
# <html lang="en">
# <head>
#   <meta charset="UTF-8">
#   <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
#   <title>Error 500 - Internal Server Error</title>
#   <link rel="stylesheet" href="/css/error.css">
# </head>
# <body>
# <h1>Error 500 - Internal Server Error</h1>
# <div id="content">
# Wooops, something went wrong
# </div>
# <div id="footer">
# Powered by <a href="http://perldancer.org/">Dancer2</a> 0.165000
# </div>
# </body>
# </html>
# '
#     expected: 'success'
# Looks like you failed 1 test of 1.
t/04-undef.t ............. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

Thanks!